Clicker/SectionControllers/PollOptionsCellSectionControllers/FROptionSectionController.swift

```diff
@@ -36,8 +36,7 @@ class FROptionSectionController: ListSectionController {
         guard let containerSize = collectionContext?.containerSize else {
             return .zero
         }
-        let cellHeight = calculateFROptionCellHeight(for: frOptionModel)
-        return CGSize(width: containerSize.width, height: cellHeight)
+        return CGSize(width: containerSize.width, height: LayoutConstants.frOptionCellHeight)
```

Comment:
Is `frOptionCellHeight` going to be the same as the cell heights for the multiple choice cells?